app/assets/javascripts/transactions.js

```diff
@@ -51,26 +46,41 @@ $(function () {
     $('#new_transaction').show();
   });
 
+  $(document).on('change', '#transfer_amount', function(e) {
+    prep_rate_and_hints(exchange_rate = false, hints = true)
+  });
+
+  $(document).on('change', '#transfer_exchange_rate', function(e) {
```

Comment:
@Gataniel let's combine this 2 events into one ` $(document).on('change', '#transfer_amount, #transfer_exchange_rate', function(e) {`